docs/docs/getting-started/components.md

```diff
@@ -73,6 +73,39 @@ Airy Core contains the following core components:
 
```

Comment:
Regarding the structure of the components page, I suggest we add the updated definition to the top of the page underneath the image. As you can see in PR #3558 I created a re-usable .mdx snippet for the definition. I would suggest adding this before we expose the cards.

Reply:
Sounds good :+1:. I will re-use the snippet from that PR :pray: